examples/resources/images/blend/embed.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2023 The Ebitengine Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package blend
+
+import (
+	_ "embed"
+)
+
+var (
+	//go:embed source.png
+	Source []byte
+	//go:embed dest.png
+	Dest []byte
```

Comment:
Dest_png

Reply:
Hm, why we're using snake case for the var name? Isn't it wrong?

Reply:
Also please add an empty line between `Source []byte` and `//go:embed dest.png`

Reply:
> Hm, why we're using snake case for the var name? Isn't it wrong?

This is just for consistency with other variable names.

Reply:
Fixed